### DISCUSSION TITLE
Welcome to Wagtail Discussions!

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#7505] [Announcements]
## 👋 Welcome!
  We’re using Github discussions as a place to connect with other members of our community and share ideas and plans for Wagtail as a project. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  We'd like to ask that any support questions ("why isn't my code working...?") are kept on our existing support channels, on [Stack Overflow](https://stackoverflow.com/questions/tagged/wagtail) and [Slack](https://github.com/wagtail/wagtail/wiki/Slack).

  To get started, comment below with an introduction of yourself and tell us about what you're doing with Wagtail.

**Comment by brylie** [comment]
Hi all! I'm Brylie from Tampere, Finland. 

I'm building an open-source [magazine publication and community website](https://github.com/WesternFriend/WF-website) with Wagtail, as a replacement to an [existing site](https://westernfriend.org) built with Drupal. We are also prototyping a [social networking website](https://github.com/brylie/wagtail-social-network) built with Wagtail.

**Comment by awhileback** [comment]
I'm Robert Clayton, I live in Texas.

I've done some backend admin deployments with Wagtail over the last couple of years for non-profit use (mainly for sending bulk emails with django-drip and django-post-office), and am currently building a media distribution system (podcasts, video, etc, like Patreon) on top of Wagtail/Django.

**Comment by saahndongransom** [comment]
Hello ,am saahndong Ransom from Cameroon. Still a beginner in django .I wish to be part of your community to share my ideas and also to learn.

**Reply by thibaudcolas** [reply]
Welcome!

**Reply by saahndongransom** [reply]
Thank you

**Comment by sdas20** [comment]
hello there Wagtail seems interesting. I am just started...On installation i get the following error: 
no such table: wagtailcore_site
Thanks for your help.

**Reply by zerolab** [reply]
Welcome! Please join the #support channel on Slack - https://github.com/wagtail/wagtail/wiki/Slack

**Comment by nirmalKumawat-kgp** [comment]
Hello Everyone , I am Nirmal Kumawat. I am new to community. Whlie joining slack workspace it is asking for spefic domains ?

**Reply by Scotchester** [reply]
Welcome, Nirmal! Slack should not require an email address with a specific domain to join. It should be open to the public. Are you getting an error of some kind?

**Reply by gasman** [reply]
@nirmalKumawat-kgp We've just updated the invitation link at https://wagtail.org/slack/ - please give it another try.

**Comment by softkeldozy** [comment]
Hello everyone, I'm Kelvin and I'm building a blog for a client using WigtailCMS

**Comment by yigakpoa** [comment]
Hello Everyone, I’m Yiga and i’m new to the community. I’ve been unable to join the slack community. I would love to contribute to the technical documentation & join the team.

**Reply by lb-** [reply]
Welcome! How are you finding Wagtail to use?

Here's a list of Documentation issues - maybe something there might be of interest for you to contribute to. 

https://github.com/wagtail/wagtail/labels/Documentation

**Comment by Yusuf-Babagana** [comment]
Hello, I am Yusuf, I am software Engineer, recently discovered wagtail and I found it amazing. Thanks

**Reply by lb-** [reply]
Awesome. Welcome.

**Comment by jainArihant12** [comment]
hello , I am wondering how can i can contribute in this organization , Guide me . Thanks

**Reply by ayaan-qadri** [reply]
You can start by reading Wagtail's official documentation for the contributing guide: https://docs.wagtail.org/en/stable/contributing/index.html

**Comment by jainArihant12** [comment]
Hello Everyone , I am new to community. Whlie joining slack workspace it is asking for spefic domains ? how can i join slack because i want to contribute .

**Reply by gasman** [reply]
Sorry about that! Please try again at https://wagtail.org/slack now - this occasionally happens because Slack places a limit on the number of people who can sign up to one invite.
